lib/src/snackbars/snackbar_position.dart

```diff
@@ -28,7 +28,7 @@ enum SnackbarPosition {
   double? bottomMargin(BuildContext context, double position) {
     return switch (this) {
       top => null,
-      bottom => position + 40 + context.bottomBarHeight,
+      bottom => position + 20 + context.bottomBarHeight,
```

Comment:
@ebabhi why you are reducing to 20?

Reply:
@kishormainali  i thought  40 was too much space bar snack bar  .. already reverted this commit